platform/server/elasticsearch/api.ts

```diff
@@ -23,20 +24,22 @@ export function registerElasticsearchRoutes(
         })
       }
     },
-    async (elasticsearch, req) => {
+    async (req, res) => {
       // WOHO! Both of these are typed!
       log.info(`field param: ${req.params.field}`);
       log.info(`query param: ${req.query.key}`);
 
       log.info('request received on [data] cluster');
 
-      const cluster = await elasticsearch.getClusterOfType('data');
+      const cluster = await service.getScopedCluster('data', req.headers);
 
-      log.info('got [data] cluster, now calling it');
+      log.info('got scoped [data] cluster, now calling it');
 
-      const response = await cluster.withRequest(req, (client, headers) =>
-        client.search({})
-      );
+      // TODO: for some reason my typescript
+      // is saying `search` isn't on this object
+      //const response = await cluster.search({});
```

Comment:
This is because `search` doesn't exist on `cluster`, only `callWithRequest`. Maybe just remove the comment and keep `const response = { hits: { total: 0 } };` below, then fix this in the follow-up?